pygsti/tools/fogitools.py

```diff
@@ -732,41 +732,6 @@ def resolve_norm_order(vecs_to_normalize, label_lists, given_norm_order):
     return (fogi_dirs, fogi_meta, dep_fogi_dirs, dep_fogi_meta)
 
 
-#def create_fogi_dir_labels(fogi_opsets, fogi_dirs, fogi_rs, fogi_gaugespace_dirs, errorgen_coefficients):
```

Comment:
I want to keep this. The FOGI code is still under active development and it isn’t immediately obvious we won’t need to revive this in the near-enough future.

Reply:
Roger that! I've reverted changes to this file.